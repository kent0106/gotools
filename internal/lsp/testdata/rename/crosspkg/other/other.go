package other

import "github.com/kent0106/gotools/internal/lsp/rename/crosspkg"

func Other() {
	crosspkg.Bar
	crosspkg.Foo() //@rename("Foo", "Flamingo")
}
