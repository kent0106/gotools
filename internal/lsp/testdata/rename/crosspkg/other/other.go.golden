-- Flamingo-rename --
crosspkg.go:
package crosspkg

func Flamingo() { //@rename("Foo", "Dolphin")

}

var Bar int //@rename("Bar", "Tomato")

other.go:
package other

import "github.com/kent0106/gotools/internal/lsp/rename/crosspkg"

func Other() {
	crosspkg.Bar
	crosspkg.Flamingo() //@rename("Foo", "Flamingo")
}

